project-tracker backend/src/app/controllers/client.ts

```diff
@@ -5,7 +5,7 @@ import { QueryBuilder } from "knex";
 import { TableNames } from "../../lib/enums";
 
 export const index = async (req: Request, res: Response) => {
-  let query: QueryBuilder = database(TableNames.clients).select();
+  let query: QueryBuilder = database(TableNames.clients).select().where({isDeleted: 'false'});
```

Comment:
softDelete-et inkább egy deletedAt mezővel szoktunk csinálni, mert az megmondja, hogy mikor törölted a rekordot. Ha nem törölted, akkor még NULL az értéke. Tehát a nem törölteket where "deletedAt is null" -lal tudod lekérni.

Reply:
persze ebből következik, hogy a deletedAt egy dátum,idővel

Reply:
timestamp nem lehetett, mert kell kezdőérték, ami nem lehet null. ha adok neki, az sem jó. így string, null default-tal, és destroy-nál az update a current time-ot teszi bele

Reply:
miért ne lehetne null a kezdő értéke a deletedAt timestamp fieldnek?

Reply:
mert nem futott le nálam a migráció. De ha segítesz, hogyan csináljam, átírom

Reply:
@pmate955 vedd át pls, ez egy gépnél egyszerűbb lesz.